components/admin_console/permission_schemes_settings/permission_system_scheme_settings/permission_system_scheme_settings.test.jsx

```diff
@@ -179,8 +177,7 @@ describe('components/admin_console/permission_schemes_settings/permission_system
         expect(wrapper.state().roles.all_users.permissions.length).toBe(0);
         expect(wrapper.state().roles.channel_admin.permissions.length).toBe(0);
         expect(wrapper.state().roles.team_admin.permissions.length).toBe(0);
-        wrapper.find('.reset-defaults-btn').first().simulate('click');
```

Comment:
I would expect to have simulate the clicks here instead of calling the `resetDefaults` directly

Reply:
Because it's a shallow render it is not longer available to this test.

Reply:
Note the shallow render no longer allows access to the confirm modal confirm button.